src/lib/config.js

```diff
@@ -2,15 +2,47 @@ import deepClone from 'deep-clone'
 import deepEqual from 'deep-equal'
 
 import stack from '../lib/stack'
-import addComingSoonApps from './comingSoon'
 
 import MENU_CONFIG from '../config/menu'
 
 const EXCLUDES = ['settings', 'onboarding']
 const CATEGORIES = ['cozy', 'partners', 'ptnb']
 
+let cachedComingSoonApps
+function fetchComingSoonApps () {
+  if (cachedComingSoonApps) return Promise.resolve(cachedComingSoonApps)
+  return stack.get.context()
+    .then(context => {
+      const comingSoonApps = context.data && context.data.attributes &&
+        context.data.attributes['coming_soon'] &&
+          Object.values(context.data.attributes['coming_soon']) || []
+
+      cachedComingSoonApps = comingSoonApps.map(app => {
+        let icon
+
+        try {
+          icon = app.slug && {
+            cached: true,
+            src: require(`../assets/icons/comingsoon/icon-${app.slug.toLowerCase()}.svg`)
```

Comment:
Why use only lowercases in names here? I would prefer folder named comingSoon and to not use `toLowerCase()` function to keep all coherent since we use camelCase everywhere. What do you think?